<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterlockDialog</class>
 <widget class="QDialog" name="InterlockDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2028</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="5" rowspan="2">
      <widget class="QListView" name="cmdList"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Конец маршрута</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="addRouteButt">
       <property name="text">
        <string>1. Добавить маршрут для выбранных сигналов</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="searchLine">
       <property name="placeholderText">
        <string>Введите текст для поиска</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="assembleButt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Собрать маршрут</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="removeCmdButt">
       <property name="text">
        <string>3. Удалить команду</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="mapConflicts">
       <property name="text">
        <string>Назначить конфликтные маршруты</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListView" name="endList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QComboBox" name="offHintCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Выключить</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Закрыть (V0)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Автоблокировка, без ограничений (Vy)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Желтый (служебн.)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Автоблокировка, ограничение  (V1)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Автоблокировка, ограничение (V2)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 желтых</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Разрешить маневры</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Разрешить меневры на св. путь.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Пригласительный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Запретить маневры</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Начало маршрута</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListView" name="beginList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="addRouteCmdButt">
       <property name="text">
        <string>3. Собрать маршрут на другой станции (развернуть перегон)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Команды</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="stationBox"/>
     </item>
     <item row="3" column="4">
      <widget class="QComboBox" name="onHintCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Выключить</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Закрыть (V0)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Автоблокировка, без ограничений (Vy)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Желтый (служебн.)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Автоблокировка, ограничение  (V1)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Автоблокировка, ограничение (V2)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 желтых</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Разрешить маневры</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Разрешить меневры на св. путь.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Пригласительный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Запретить маневры</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Сигналы</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" rowspan="2">
      <widget class="QListView" name="sigList">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QTreeView" name="trackView">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="addSigButt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>3. Добавить команду на открытие сигнала</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Траектории, стрелки</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="addTrajsButt">
       <property name="text">
        <string>2. Задать траектории для проверки на занятость</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="addJButt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>3. Добавить команду на переключение стрелки</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="removeTrajButt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>2. Удалить выбранную траекторию из списка</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="sideBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Отклонение</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="removeRouteButt">
       <property name="text">
        <string>Удалить маршрут</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="2">
      <widget class="QListView" name="trajsList">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
